standalone/inc/pup/PUPScreen.cpp

```diff
@@ -459,15 +459,30 @@ void PUPScreen::QueueBG(int mode)
 
 void PUPScreen::QueueTrigger(char type, int number, int value)
 {
-   if (value == 0)
-      return;
+   // we store the whole state to be able to mach later
+   const std::string name = std::string(1, type) + std::to_string(number);
+   m_triggersState[name] = value;
 
+   // The first trigger is the main trigger, the rest we just check
    vector<PUPTrigger*>* pTriggers = GetTriggers(type + std::to_string(number));
    if (!pTriggers)
       return;
 
    for (PUPTrigger* pTrigger : *pTriggers) {
-      PUPTriggerRequest* pRequest = new PUPTriggerRequest();
+      if (!pTrigger->IsActive())
+         continue;
+
+      for (const auto& [m_sName, value] : pTrigger->GetTriggers()) {
+         auto currentValue = m_triggersState.find(m_sName);
+         if (currentValue == m_triggersState.end()) {
```

Comment:
for formatting consistency, single line if's you don't need braces unless you have another large if, for, or are using PLOG statements. also, it looks like you could have just OR'd it

```
if (currentValue == m_triggersState.end() || currentValue->second != value)
   return;
```

Reply:
Can such things be put in the clang-format file?